### PR TITLE
Setting up microsite on Github Pages

### LINKED ISSUE #34: Set up Github pages microsite

### PR DESCRIPTION
**Pull request by suzubara**
## Description 

We're publishing this branch to our Github pages microsite: https://ussf-orbit.github.io/ussf-portal/

Use this branch/PR for adding initial content & setup.
- Right now it's using the "Minimal" Jekyll theme. Supported themes are listed here: https://pages.github.com/themes/
- Feel free to push additional content to this branch!

Fixes #34

### CODE REVIEW

**Comment by noahfirth** [comment]
I'm going to toss some more content on here. I'm just going to commit right to this branch and we can make any edits or changes as we go. I'd rather get this pushed up and published knowing it's a work in progress. Going forward, do we want to keep this branch open for content changes or do we want to create a new branch every time we want to add or change content?

**Review by noahfirth — APPROVED** [review]
I'd love to get this merged and see how it looks when we have it live. We can keep 34-microsite as a branch and use it for content changes in the future. Do we want to look at having this branch auto merge in the future?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ## About
 Central hub for USSF Portal
 
+## USSF Portal Roadmap
+* [Roadmap](https://ussf-orbit.github.io/ussf-portal/ussf-portal-roadmap)
```
Comment (esacteksab) [inline README.md:5]:
```suggestion
* [Roadmap](https://ussf-orbit.github.io/ussf-portal/USSF-Portal-Product-Roadmap)
```
The link in the PR 404's. This is the correct link based on the document below.

File: USSF-Portal-Product-Roadmap.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+layout: page
```
Comment (esacteksab) [inline USSF-Portal-Product-Roadmap.md:1]: Looking at the preview https://ussf-orbit.github.io/ussf-portal/USSF-Portal-Product-Roadmap the tables don't seem to be properly formatted and the emojis aren't coming across.

Comment (noahfirth) [reply USSF-Portal-Product-Roadmap.md:1]: You're right, let me look at the GitHub pages documentation and figure out the table layouts.

File: USSF-Portal-Product-Roadmap.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+title: "USSF Portal Roadmap"
+permalink: /ussf-portal-roadmap/
```
Comment (esacteksab) [inline USSF-Portal-Product-Roadmap.md:3]: Based on the title of the page, I think this should be `USSF-Portal-Product-Roadmap` instead
```suggestion
permalink: /USSF-Portal-Product-Roadmap/
```

**Pull request merged**